app/src/App.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div id="main" class="min-h-screen w-screen">
+    <div id="card" class="bg-black h-full w-full">
+      <h1 class="mb-4 bg-white">Match 3 ⛄ to win</h1>
+      <div id="game" class="h-full w-full bg-black grid grid-cols-4 grid-rows-3 gap-y-4 gap-x-1">
+        <ScratchBlock v-for="i in 12" :key="i" /> 
+      </div>
+    </div>
+  </div>
+</template>
+
+
+<script>
+import ScratchBlock from "./components/ScratchBlock.vue";
+
+export default {
+  components: {
+    ScratchBlock,
+  },
+  mounted() {
+    this.$nextTick(() => {
```

Comment:
this and the load event listener are required due to the width calculation on the scratch canvas elements. If this isn't done then the widths are too large as they're being calculated too soon.

Reply:
Also, currently, if the window is resized after the widths have been calculated then the existing widths will still be in use. This can be looked into.

Reply:
^ This has since been addressed